_data/udayaravelli/udayaravelli.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Uday Aravelli
+institution: Gitam Institute of Technology and Management
+profile_pic: Uday.png 

```

Comment:
Incorrect file name, please do one of the following:
1. Change `Uday.png` to `Uday.jpg` in the markdown file.
2. Rename `Uday.jpg` to `Uday.png`.
I'd suggest you make the change in the markdown file!